composer.json

```diff
@@ -14,14 +14,17 @@
     ],
     "require": {
         "php": ">=7.4",
-        "behat/mink": "^1.9@dev"
+        "ext-json": "*",
+        "behat/mink": "^1.9@dev",
```

Comment:
this should be changed to `^1.11@dev` to support passing Throwable in the previous exception.